src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -4133,6 +4371,43 @@ export class ResultsViewPageStore {
         },
     });
 
+    readonly getOncoKbStructuralVariantAnnotationForOncoprint = remoteData<
```

Comment:
seems like mostly we don't name these mobxpromises "get..."   which implies a function?